protocol/dubbo/poolqueue.go

```diff
@@ -0,0 +1,181 @@
+package dubbo
```

Comment:
This is a inner package in golang, Maybe we can move it to dubbogo/gost.

Reply:
yes, here just for easy review. I will move it to dubbogo/gost after review.